Jellyfin.Server.Implementations/Extensions/ServiceCollectionExtensions.cs

```diff
@@ -73,6 +74,7 @@ public static IServiceCollection AddJellyfinDbContext(
                 efCoreConfiguration = new DatabaseConfigurationOptions()
                 {
                     DatabaseType = "Jellyfin-SQLite",
+                    LockingBehavior = DatabaseLockingBehaviorTypes.NoLock
```

Comment:
Same as above

Reply:
The default No-Lock behavior works for most users and is the most performant, sqlite internally __should__ be performing a locking behavior in its own, yet some users still have problems with it therefor its configurable.